### ISSUE TITLE
Route Table Inquiry

### ISSUE THREAD

**Issue by crisarceramos** [daniel-rhoades/acs-engine-terraform#9]
Hi Daniel, 
Thanks for this tutorial. Just want to confirm, Route Table creation is not included in the tutorial right? Coz the az resource list is returning nothing. So we have to manually create it?

Cheers,
Cris

**Comment by jgrossmac** [comment]
Yes it appears you need to create the route table as well.  You will also need to make sure your command for null_resource.fix_routetable outputs in json.
`az resource list --resource-group <RESOURCE_GROUP> --resource-type Microsoft.Network/routeTables --output json | jq -r '.[] | .id' `

**Comment by jgrossmac** [comment]
My route table configs looks like this:

> // Azure Route Table
resource "azurerm_route_table" "default" {
  name                          = "${azurerm_virtual_network.default.name}_rt"
  location                      = "${var.region}"
  resource_group_name           = "${var.cluster_name}-${random_integer.default.result}"
  disable_bgp_route_propagation = false
  depends_on                    = ["azurerm_virtual_network.default"]

 > tags {
    Cluster     = "${var.cluster_name}-${random_integer.default.result}"
    Environment = "${terraform.workspace}"
    ManagedBy   = "Terraform"
  }
}
